### DISCUSSION TITLE
how to override isDraggable prop of GridItem

### DISCUSSION THREAD

**Discussion by mkamalkayani** [react-grid-layout/react-grid-layout#1510] [Q&A] [answered]
Currently, If a grid item has a isDraggable prop, it overrides the isDraggable or static prop it gets from the GridLayout. Is there a way to prioritise the isDraggable prop of GridLayout over the individual grid items?

**Comment by STRML** [comment, marked as answer]
No, the intention is to do the opposite. The more specific option overrides
the general one.

**Reply by mkamalkayani** [reply]
Thanks.
